Build from source fails: file cpan/engine/read_only.time-stamp missing

When Marpa::R3 is installed from Git, the Makefile fails:

``` console
$ make
cd engine && make
make[1]: Entering directory '$MARPA_R3/cpan/engine'
make[1]: *** No rule to make target 'read_only.time-stamp', needed by 'gnu_ac_build.time-stamp'.  Stop.
make[1]: Leaving directory '$MARPA_R3/cpan/engine'
Makefile:1390: recipe for target 'engine/gnu_ac_build/.libs/libmarpa.a' failed
make: *** [engine/gnu_ac_build/.libs/libmarpa.a] Error 2
```

In the tarball uploaded to CPAN, a file `cpan/engine/read_only.time-stamp` is present. Should this file be checked in, or should the Makefile contain a rule to generate this file?

Workaround: `date >cpan/engine/read_only.time-stamp`

This error could have been detected with an automated build, e.g. via Travis CI. Would there be interest in a pull request to add the necessary config?

--- a/cpan/engine/Makefile.PL
+++ b/cpan/engine/Makefile.PL
@@ -92,6 +92,11 @@ undef &MY::postamble; # suppress warning
       );
 
     push @postamble_pieces, <<'END_OF_POSTAMBLE_PIECE';
+read_only.time-stamp:
+	date > $@
+END_OF_POSTAMBLE_PIECE
+
+    push @postamble_pieces, <<'END_OF_POSTAMBLE_PIECE';
 gnu_ac_build.time-stamp: read_only.time-stamp
 	$(RM_RF) gnu_ac_build
 	$(LIBMARPA_INSTALL) read_only gnu_ac_build

